pkg/manager/impl/util/filters.go

```diff
@@ -120,6 +120,54 @@ func prepareValues(field string, values []string) (interface{}, error) {
 	return preparedValues, nil
 }
 
+var allowedJoinTableColumns = map[common.Entity]map[common.Entity]sets.String{
+	common.Execution: {
+		common.Execution:  gormimpl.ExecutionColumns,
```

Comment:
instead of duplicating the mapping between common.Entity -> allowedcolumns, can we create two sets here?
1) for entity -> allowed join entities
2) for entity -> allowed columns